site/pages/institution-fr.hbs

```diff
@@ -8,21 +8,19 @@
 		{ "title": "Accueil", "link": "https://www.canada.ca/fr.html" },
 		{ "title": "Ministères et organismes", "link": "https://www.canada.ca/fr/gouvernement/min.html" }
 	],
-	"dateModified": "2014-10-24",
+	"dateModified": "2017-09-29",
 	"share": "true"
 }
 ---
 <div class="row">
-	<div class="col-sm-7 pull-left">
-		<p>Nous sommes une institution fédérale qui fait partie du portefeuille du <a href="#">[nom du portfolio]</a>.</p>
-		<p>[Nom de l’institution] Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras sagittis massa sed libero eleifend, ac iaculis urna tempor. Cras sagittis massa sed libero eleifend, ac iaculis urna tempor.</p>
-	</div>
+<div class="col-sm-7 pull-left">
+<p>[Nom de l’institution] Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras sagittis massa sed libero eleifend, ac iaculis urna tempor. Cras sagittis massa sed libero eleifend, ac iaculis urna tempor.</p>
+</div>
 	<div class="clr-rght-sm clr-rght-md clr-rght-lg"></div>
 	<div class="col-sm-5 pull-right">
```

Comment:
Can you do that block without the "pull-". Try to use instead "col-sm-pull-" and "col-sm-push-"